extensions/parquet/table/plugin/channelprovider/s3/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+dependencies {
+    api project(':extensions-parquet-base')
```

Comment:
Ideally, we'd be able to remove direct dependency on parquet.

My motivation is to be able to re-use this work with S3 CSVs. Should be easy to test out, `io.deephaven.csv.CsvTools#readCsv(java.io.InputStream)` + `java.nio.channels.Channels#newInputStream(java.nio.channels.ReadableByteChannel)`.

Reply:
Agreed, right now we need it for the `SeekableChannelsProvider` and `SeekableChannelsProviderPlugin` interfaces. Do you think we should move them somewhere else?